docs/content/latest/deploy/kubernetes/clients.md

```diff
@@ -109,3 +109,137 @@ Status:
         router: no errors
         loadbalancer emulator: no errors
 ```
+
+## Connecting TLS Secured YugabyteDB cluster deployed by Helm Charts
+
```

Comment:
We should link to the helm deployment doc from here and say `To bring up a yugabyte server with encryption in transit (TLS) enabled, set the flag tls.enabled = true in the helm overrides`

Reply:
Okay, I'll update the suggested changes.